src/qml/RelationCombobox.qml

```diff
@@ -334,102 +344,210 @@ Item {
     }
 
     Rectangle {
-      id: searchableLabel
-      visible: !comboBox.visible
-      height: fontMetrics.height + 10
-      Layout.fillWidth: true
-      Layout.topMargin: 5
-      Layout.bottomMargin: 5
+        id: searchable
```

Comment:
Is it possible to keep the whole file the same level of indentation. I see it is 2 whitespaces above, it is 4 here.

Reply:
4 is the default of qmlformat, shall we just go with that?

Reply:
I'll apply it to RelationCombobox.qml when everything else's been fixed, otherwise it'll be a nightmare for people to review.